fluent-plugin-detect-exceptions.gemspec

```diff
@@ -25,4 +25,5 @@ eos
   gem.add_development_dependency 'rake', '~> 10.3'
   gem.add_development_dependency 'rubocop', '= 0.42.0'
   gem.add_development_dependency 'test-unit', '~> 3.0'
+  gem.add_development_dependency 'flexmock', '~> 2.0'
```

Comment:
How about use rr for mocking via test-unit-rr?
It is transferred project and not marked as forked repository: https://github.com/rr/rr